content/en/docs/architecture/release-gating.md

```diff
@@ -67,88 +36,215 @@ Good candidates include:
+to tie into OpenShift's CI and block releases. For example, OCS is a part of the OpenShift platform,
+and if an upgrade breaks OCS, we have broken part of our platform. Extending our testing to
+these components is a requirement to perform regression and early verification.
 
 {{< alert title="Note" color="info" >}}
 OpenShift CI creates thousands of clusters a week - we can certainly afford to run a few more tests. However, we want to
 spread that investment to derive the maximum benefit for everyone, so having OpenShift CI run 30 variations of your
 exact same tests in slightly different configurations 4 times a day may not be the best way to test.
 {{< /alert >}}
 
-### An Introduction to Aggregated Jobs
+### Pre-merge testing
+
+Detecting a regression before it ends up in the release payload at all is the ultimate 
+form of release testing. It is the most expensive form because each 
+change is tested in isolation. Additionally, due to the scope of OpenShift and it's 
+configurations, it is not possible to test everything against every code change.
+
+Repository owners should make an educated selection of the most impactful jobs that 
+have a high likelihood of detecting a problem. As the team learns, this list should 
+be modified. For example, if a change to the repository causes an incident (see below),
+the job that was affected should become a presubmit job on that repository.
+
+We generally recommend you have coverage for both the parallel conformance and
+serial suites, as well as an upgrade job.
+
+Typically, jobs which are present on many repositories that span multiple teams
+should also be payload blocking.
+
+### Payload blocking
+
+Roughly every six hours, we produce a release payload which is a roll-up of all the 
+changes that have merged since the previous one. Each payload runs a set of blocking
+CI jobs that must succeed for the payload to be accepted. Our goal is to achieve an
+accepted nightly payload every day.
+
+Payload blocking jobs are monitored by the [technical release team](https://docs.ci.openshift.org/docs/release-oversight/the-technical-release-team/),
+and regressions to blocking jobs trigger an incident. These jobs _must never break_.
+Regressions to these jobs are expected to be detected and reverted within hours. "Fix
+forward" is not allowed, the identified change must be reverted and reintroduced with
+payload testing confirming the fix.
+
+Due to the disruptive nature of reverts, good candidates are jobs that have wide 
+impacts if they break -- core functionality on key platforms of the product, or jobs 
+that run as presubmits on a diverse set of repositories.
+
+{{< alert title="Note" color="info" >}}
+TRT will typically not see or revert changes that break jobs which are not payload
+blocking. For changes which break these jobs, the decision to revert or fix will need
+to be handled between the affected team and the team that made the change.
+{{< /alert >}}
+
+
+### Component readiness
```

Comment:
indicate the source of job runs and fact that select set of business owners choose which combinations to enforce regression protection on.